modules/bmo-performing-a-live-update-to-the-hostfirmwaresettings-resource.adoc

```diff
@@ -2,22 +2,27 @@
 //
 // * installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc
 :_mod-docs-content-type: PROCEDURE
-[id="bmo-patching-the-hostfirmwaresettings-resource_{context}"]
-= Patching the HostFirmwareSettings resource
+[id="bmo-performing-a-live-update-to-the-hostfirmwaresettings-resource_{context}"]
+= Performing a live update to the HostFirmwareSettings resource
 
-To patch the `HostFirmwareSettings` for a provisioned host, perform the following actions:
+You can perform a live update to the `HostFirmareSettings` resource after it has begun running workloads. Live updates do not detach the host from Ironic and reattach it after performing the update. To perform a live update to `HostFirmwareSettings`, take the following actions:
 
+* Update the resource.
```

Comment:
Edit or patch the HostFirmwareSettings Custom Resource
Drain the host.
Power off the host for approximately 5 minutes
Power the host back on
Once the operation is successful, uncordon the host so it can re-join the cluster